ubuntudesign/www.ubuntu.com
# add autopilot ebook to autopilot instruction pages

## Done

* added details of a new ebook with a form onto the two autopilot instruction pages
* created a [marketo form](https://app-sjg.marketo.com/#PG1550A1) to capture readers details
* created a new partial for the strip
* added ajax to background submit the form and kick off the download
* updated other text on the instructions and /cloud/openstack/autopilot page to make the journey's clearer and easier
* removed the 'ask an expert' cta on the instructions and added a cloud picto

## QA

1. go to /cloud/openstack/autopilot and see that the main CTA is the simplier 'Get Autopilot' - [copy doc](https://docs.google.com/document/d/1VmFtp_gEYvioRgGXbiQZpVGBhlJoIZAirLk6IOVUjUE/edit)
2. go to /download/cloud/install-openstack-with-autopilot and see the new strip and [copy updates](https://docs.google.com/document/d/1aoRSMJkGsAacGoLlIuzPg3J-630OVhAEORNEa2U5UJ8/edit#heading=h.yb98vv47gv6p)
3. try the form with good and bad data
4. see that the form submits with good data and redirects to ubuntu.com//download/cloud/install-openstack-with-autopilot#instructions (note: the internal link will not work on a demo)
5. see that the user was added to [marketo](https://app-sjg.marketo.com/#PG1550E5)
6. go to /download/cloud/install-autopilot-testdrive and see the new strip and [copy updates](https://docs.google.com/document/d/1P-Tu-TELbc9iOUczIgQg7ATLfciE0jLnvteMsU1C3nc/edit)
7. try the form with good and bad data
8. see that the form submits with good data and redirects to ubuntu.com//download/cloud/install-autopilot-testdrive#instructions (note: the internal link will not work on a demo)
9. see that the user was added to [marketo](https://app-sjg.marketo.com/#PG1550E5)

## Issue / Card

[trello card 1](https://trello.com/c/MGZ7hO7A)
[trello card 2](https://trello.com/c/HgZdLQUi)

## Screenshots

![image](https://cloud.githubusercontent.com/assets/441217/17167850/3147f6be-53d7-11e6-94e3-31f2dc6dab22.png)
